client/public/index.html

```diff
@@ -24,6 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="%PUBLIC_URL%/env-config.js"></script>
```

Comment:
Change the variable name `PUBLIC_URL` to something more descriptive. For example `APP_URL`? Using the keyword PUBLIC does not sound right.

Reply:
`PUBLIC_URL` is not my variable name, it is the name of a constant [provided by React to provide access to statically-served assets](https://create-react-app.dev/docs/using-the-public-folder/#adding-assets-outside-of-the-module-system)